Automattic/jetpack
# Protect: Update vulnerability list designs

#### Description
This PR implements the necessary design changes to the Protect vulnerability list, as part of the Protect and Scan integration project.

#### Changes proposed in this Pull Request:

* Adds a call to action to each threat item dropdown
* Adds analytics tracking for the click event

#### TODO items:
* Update required plan, when available

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1203041610910920
- p1HpG7-gSG-p2

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:





* Checkout this branch
* Run your Jurassic Tube
* Activate Protect
* Proceed with a Free connection and allow the initial report to generate
* From the corresponding status page view the dropdown for each identified threat item and compare it against Figma designs
* Enable mobile view and compare
* Ensure the CTA responds as expected
